tests/travis/install-mssql-cs.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo Setting up Microsoft SQL Server
+
+sudo docker pull microsoft/mssql-server-linux:2017-latest
+sudo docker run \
+    -e 'ACCEPT_EULA=Y' \
+    -e 'SA_PASSWORD=Doctrine2018' \
+    -e 'MSSQL_COLLATION=Latin1_General_100_CS_AS' \
```

Comment:
This could be moved to the default script where the collation is defined as `MSSQL_COLLATION=${MSSQL_COLLATION:-Latin1_General_100_CI_AS}` and can be overridden in `.travis.yml`.

Reply:
@morozov I think you better take it into your own hands. I'm not quite sure how to parameterize Travis's tests to be ok for you.